### ISSUE TITLE
webserver hangs in v5 datalogger

### ISSUE THREAD

**Issue by AndreaBorgia-Abo** [stanleyhuangyc/Freematics#52]
Up to, say, a couple of months ago the internal webserver in datalogger worked ok and would be available both in AP and in station mode.

With today's build, the connection hangs and no output is returned:
`Motion:0.58`
`Wakeup`
`File:/DATA/71.CSV`
`OBD...NO`
`Standby`
`ACC Bias:0.00/-0.00/-1.27`
`[56] Client IP: 192.168.4.2`
`[56] Socket closed, 0 connections`

This happens also when only AP mode is enabled; tested with Firefox, wget and also raw telnet.
Can anyone else verify?

**Comment by AndreaBorgia-Abo** [comment]
This is still happening with today's commit.  Wanted to "git bisect" it but versions prior to commit 4a4a2d30171796498f7848655e8a10e39d9faee6 fail to build for a deprecated bta_api.h reference.

**Comment by AndreaBorgia-Abo** [comment]
Found it: first bad commit is f8e709ac003a26dc9e1945af52838ee8c46bdcdb

I can provide log file and also config.h / platformio.ini for each step, since each step used the most up-to-date platform available when the corresponding commit was logged.

**Comment by AndreaBorgia-Abo** [comment]
[Escalating the issue to Platform](https://community.platformio.org/t/webserver-hangs-with-espressif32-v1-1-0-works-fine-with-v1-0-2/4780) first and, perhaps, ESP32 as well.

**Comment by AndreaBorgia-Abo** [comment]
After the first and so far only merge of a [PR](https://github.com/stanleyhuangyc/Freematics/pull/76) on this project, I wondered if it applied to my bug as well.

To test, I rebuilt f8e709ac003a26dc9e1945af52838ee8c46bdcdb first as is and verified the bug was still reproducible: it was. Then I applied the patch, rebuilt and tried again: bingo, now the webserver actually replies.

Closing this and related tickets.
